src/server/views/amsapp-stadspas-administratienummer.pug

```diff
@@ -23,6 +23,32 @@ html(lang="nl")
 
       a(href=appHref id='protocol-link-button') Open Amsterdam App
 
-    script(type='text/javascript' nonce=nonce).
+    script(type='text/javascript' nonce=nonce).       
+      const logoutUrl = '#{logoutUrl}';
+
+      async function performLogout() {
+        console.log('performLogout');
+        try {
+          const response = await fetch(logoutUrl, {
+            method: 'GET',
+            credentials: 'include'
+          });
+        } catch (error) {
+          console.error('Error during logout:', error);
+        }
+      }
+
+      async function logoutAndOpenApp() {
+        await performLogout();
         document.getElementById('protocol-link-button').click();
+      }
+
+      document.addEventListener('DOMContentLoaded', () => {
+        setTimeout(logoutAndOpenApp, 0);
```

Comment:
Die setTimeout is toch niet nodig? Er is verder geen enkele code execution in deze pagina.

Reply:
klopt idd